portal/gatsby-node.js

```diff
@@ -1,6 +1,7 @@
 exports.onCreatePage = async ({ page, actions }) => {
     const { createPage, deletePage } = actions;
-    const componentPageMatch = page.path.match(/\/.*react.*\/documentation\/(.*)\//);
+    const componentPageMatch = page.path.match(/\/(.*(react|util).*)\/documentation\/(.*)\//);
```

Comment:
Tror vi trenger en ny regel under for å håndtere de nye pakkenavnene som blir tatt med. Denne endringen alene vil endre adressen til alle komponentsidene, slik at vi får en haug med døde lenker rundt omkring 😅

Ønsket oppførsel vil være at den genererer path ut fra den delen av pakkenavnet som _ikke_ angir type (f.eks. `-react` eller `-util`), slik at `button-react` blir til `button`, og `validation-util` blir til `validation`.

Reply:
Blir konflikt med colors branchen, løst litt aneledes der, så kan evt fikse det når den merges om denne går først.